lib/model_probe.rb

```diff
@@ -213,4 +214,11 @@ def probe_indexes
   def probe_footer
     puts Color.gray "".ljust(110, "=")
   end
+
+  def connection_port_from(config, default: 5432)
+    return default unless config.key?(:port)
+    return default if config[:port].nil? || config[:port].empty?
+
+    config[:port]
+  end
```

Comment:
We probably shouldn't assume Postgres is default given all the other database engines that Rails can work with. Let's update to return nil when port is not configured.

Reply:
Thank you for reviewing this change.

I removed the default, and so `connection_port_from` will return nil when the port is not configured. 

It seems to work fine without extra conditions in the `ddl` method as `pg_dump --port=` will use the default 5432 when `--port=` is empty